crates/astria-conductor/src/executor/mod.rs

```diff
@@ -214,6 +231,37 @@ impl<C: ExecutionClient> Executor<C> {
             return Ok(Some(execution_hash.clone()));
         }
 
+        let mut final_response = Ok(Some(self.execution_state.clone()));
+        // TODO: (GHI 250 - https://github.com/astriaorg/astria/issues/250):
+        // if blocks are coming from DA, do we still need them to pass through
+        // the queue? if not, we can skip the insert and pop_blocks calls
+        if let Ok(None) = self.block_queue.insert(block.clone()) {
+            return final_response;
+        };
+        // TODO: (GHI 250 - https://github.com/astriaorg/astria/issues/250):
+        // add a match statement here to either `pop_blocks` (returns soft and
+        // head) or `pop_soft_blocks` (just soft) based on the `execution_commit_level` setting
+        let queued_blocks = self.block_queue.pop_blocks();
+
+        if let Some(blocks) = queued_blocks {
+            // execute all the blocks returned from the queue
+            for block in blocks {
+                final_response = self.execute_single_block(block).await;
+            }
+        }
```

Comment:
this ignores/overwrites the error if one of these blocks fails to execute, please check err after every call